src/aux-server/aux-web/aux-player/interaction/DragOperation/PlayerFileDragOperation.ts

```diff
@@ -24,6 +25,7 @@ import { InventorySimulation3D } from '../../scene/InventorySimulation3D';
 import { PlayerGame } from '../../scene/PlayerGame';
 import { VRController3D } from '../../../shared/scene/vr/VRController3D';
 import { differenceBy, take, drop } from 'lodash';
+import { GridTile } from 'aux-web/aux-player/PlayerGrid3D';
```

Comment:
@shaneYeti Do we need this import? If so, can the path be rewritten to use a relative file path? (otherwise the compiler will complain about not being able to find the file)